recipe/meta.yaml

```diff
@@ -26,11 +26,11 @@ requirements:
   host:
     - python
     - setuptools
-    - numpy >=1.10
+    - numpy 1.11
+    - cycler 0.10
```

Comment:
Does this one have to be that strict? Can do `>=0.10`?

Reply:
I think so. I also think many of the host dependencies are not necessary. matplotlib isn't very good at seperating "build time" and "run time" dependencies.